app/build.gradle

```diff
@@ -1,59 +1,89 @@
 buildscript {
-    repositories {
-        maven { url 'https://repository.exoplatform.org/content/repositories/fabric-releases/' }
-    }
-
     dependencies {
-        classpath 'io.fabric.tools:gradle:1.21.6'
+        classpath 'io.fabric.tools:gradle:1.22.1'
     }
 }
 
 apply plugin: 'com.android.application'
 apply plugin: 'io.fabric'
 
-repositories {
-    maven { url 'https://repository.exoplatform.org/content/repositories/fabric-releases/' }
-}
-
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.3"
+    compileSdkVersion rootProject.ext.compileSdkVersion
 
     defaultConfig {
         applicationId "org.exoplatform"
-        minSdkVersion 16
-        targetSdkVersion 27
-        versionCode 1604281805
-        versionName "5.0"
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.compileSdkVersion
+        versionCode 27
+        versionName "5.0.0-SNAPSHOT"
+        manifestPlaceholders = [
+            crashlyticsApiKey: "b8421f485868032ad402cef01a4bd7c70263d97e",
+            appLabelEnvironmentSuffix:""
+        ]
+    }
+    signingConfigs {
+        release
     }
     useLibrary 'org.apache.http.legacy'
+
     buildTypes {
         release {
             minifyEnabled false
+            useProguard false
+            signingConfig signingConfigs.release
```

Comment:
I have this error :

> Error:Execution failed for task ':app:validateSigningBetaRelease'.
Keystore file not set for signing config release

Reply:
which command do you type to have this error ?

Reply:
I just try to run the app in Android Studio (launch the emulator).

Reply:
You should not use the betaRelease variant but the localDebug one.